workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/WorkflowInterceptor.kt

```diff
@@ -172,22 +171,21 @@ public interface WorkflowInterceptor {
    * A render pass has been skipped by an optimization, multiple actions are applied before
    * the runtime produces a rendering.
    */
-  public object RenderPassSkipped : RuntimeUpdate
+  public data object RenderPassSkipped : RuntimeUpdate
```

Comment:
We avoid using `data` in public API, remember? Seems like this would be fine as just `object` / `class`.

Reply:
The arguments for not using data classes in public API don't apply to objects, since they contain no data by definition.

And the data class was previously a value class, which is even more impossible to change than a data class, so this is no more constrained.